src/languages/yaml.js

```diff
@@ -20,12 +20,12 @@ export default function(hljs) {
   const KEY = {
     className: 'attr',
     variants: [
-      // added brackets support 
-      { begin: /\w[\w :()\./-]*:(?=[ \t]|$)/ },
-      { // double quoted keys - with brackets
-        begin: /"\w[\w :()\./-]*":(?=[ \t]|$)/ },
-      { // single quoted keys - with brackets
-        begin: /'\w[\w :()\./-]*':(?=[ \t]|$)/ },
+      // added brackets and other special characters support
+      { begin: /\w[\w\s:/.\-()]*:(?=[ \t]|$)/ },
```

Comment:
You just changed the order - other than `' '` and `\s`... why?

And is \s REALLY what we want?  That can include tabs and carriage returns... is there a reason a literal space we not sufficient?

Reply:
@joshgoebel I think I got it wrong. I changed it to just add * and @, what do you think?